template.sh

```diff
@@ -23,7 +23,7 @@ replace_all () {
   replace "$filename" '\]\]' '}}'
 }
 
-for i in aws azure common kvm vmware; do
+for i in aws azure common vmware; do
```

Comment:
Folder was removed some time previously

Reply:
What's the story with kvm? There is still a `policies-kvm` folder

Reply:
Not sure - I can't find the commit which removed it. Removed the helm folder as well now :D

Reply:
I added it to the list when I was working on the script (didn't want to feel left out), but we don't have any KVM-specific policies so it makes sense to drop it.